explainers/small-deliveries.html

```diff
@@ -42,7 +42,7 @@
         color: rgba(52, 73, 94, 1.0);
         text-decoration: none;
         font-size: 0.7em;
-        padding: 4px 10px;
+        padding: 12px 10px;
```

Comment:
This seems like a really big change to the button design. Why is it here? What is it you are trying to achieve?